### PR TITLE
Implement changing theme feature using Dropdown menu

### PR DESCRIPTION
**Pull request by korneliakobiela**
[Issue] #1131
[Problem] There wasn't possibility to change theme
[Solution] Implement dropdown menu with changing theme feature.

[NOTE] I think there is something wrong with DropDown menu.
I couldn't find wat to make more menu button the placeholder of menu.

Signed-off-by: Kornelia Kobiela <korneliak.95@gmail.com>

### CODE REVIEW

**Comment by lslachciak** [comment]
@korneliakobiela in your PR user is able to select theme after using additional menu
I think this breaks original UI.
Screenshot from 2020-05-27 08-55-33

As I wrote in https://github.com/Samsung/TAU/issues/1131 user should be able to select theme after clicking three dots. This will not break UI look.

**Comment by lslachciak** [comment]
Example of three dots submenu you can see in
 /mobile/UIComponents/components/appbar/more-option-list-max-width.html

File: examples/mobile/ScheduleApp/index.html [hunk 1]
```diff
@@ -26,6 +26,15 @@
 			<div class="ui-appbar-action-buttons-container">
 				<a class="ui-btn ui-btn-icon ui-btn-icon-add" data-style="flat" href="#turn-on-dialog"></a>
 				<button class="ui-btn ui-btn-icon ui-btn-icon-more" data-style="flat"></button>
+				<select data-native-menu="false" id="theme-selector" class="app-hidden">
+					<option value="choose-one" data-placeholder="true">Theme</option>
+					<option value="light">
+						Light theme
```
Comment (lslachciak) [inline examples/mobile/ScheduleApp/index.html]: please be consistent in capitals for Theme
Light **t**heme
vs
Dark **T**heme

**Review by lslachciak — CHANGES_REQUESTED** [review]
Requested: Minor fix in code and menu position change

File: examples/mobile/ScheduleApp/index.html (hunk 1 above)
Comment (korneliakobiela) [reply examples/mobile/ScheduleApp/index.html]: Done!

**Review by lslachciak — APPROVED** [review]

**Pull request merged**